build.gradle

```diff
@@ -58,18 +58,45 @@ repositories {
     mavenCentral()
 }
 
-dependencies {
-	compile 'com.microsoft.azure:azure-keyvault:0.9.7'
-	
-	testCompile 'junit:junit:4.12',
-	'org.junit.platform:junit-platform-console:1.0.0-M3',
-	'org.junit.platform:junit-platform-commons:1.0.0-M3',
-	'org.junit.platform:junit-platform-engine:1.0.0-M3',
-	'org.junit.platform:junit-platform-launcher:1.0.0-M3',
-	'org.junit.platform:junit-platform-runner:1.0.0-M3',
-	'org.junit.platform:junit-platform-surefire-provider:1.0.0-M3',
-	'org.junit.jupiter:junit-jupiter-api:5.0.0-M3',
-	'org.junit.jupiter:junit-jupiter-engine:5.0.0-M3'
+//enable junit gradle plugin
+buildscript {
+	repositories {
+		mavenCentral()
+
+	}
+	dependencies {
+		classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.0-M3'
+
+	}
 }
 
+apply plugin: 'org.junit.platform.gradle.plugin'
 
+ext.junit4Version        = '4.12'
+ext.junitPlatformVersion = '1.0.0-M3'
+ext.junitJupiterVersion  = '5.0.0-M3'
+
+junitPlatform {
+	filters {
+		tags {
+			// comment out the exclude property to run all tests
+			exclude 'slow'
```

Comment:
by default it will skip all the testcases with tag 'slow'. As discussed, we can come up with better alternative after we finalize on build automation system.